### DISCUSSION TITLE
飢餓會造成什麼

### DISCUSSION THREAD

**Discussion by Ryanlolry** [1101-GS4719-C/Python-Community-GS4719-C#1751] [General]
人類能在不進食下維持一定的身體機能長達7天，但如果一天不進食後，細胞便會開始消耗肌肉的能量，使儲存在身體的能量降低，若在同時還進行高消耗度的活動，如上課，回應很爛的發言等，便會加速能量的損失，讓人神智不清，所以在進食時便會過度，所以要避免長時間不進食，因為未影響到身體機能，且不進食後，在維持一定身體機能下，會去分解脂肪或肌肉組織，降低身體的強度。

**Comment by Patsy17** [comment]
飢餓會造成我心情不好：（

**Comment by poiu6fi** [comment]
肚子餓的時候都會情緒很暴躁，想專心做事也很難，還是填飽肚子最重要

**Comment by Tchang323** [comment]
飢餓好像比肚子餓還嚴重些。我自己是還沒體會過飢餓的感覺，但是肚子餓的話，確實會影響情緒、專注力降低，也會讓我食欲大增。所以最好還是要有規律飲食，讓身體吸收能量
